scripts/contractInteraction/staking_vesting.py

```diff
@@ -366,9 +366,9 @@ def updateLockedSOV():
     # sendWithMultisig(conf.contracts['multisig'], lockedSOV.address, data, conf.acct)
 
 #receiver is usually the multisig
-def governanceWithdrawVesting( vesting,  receiver):
+def governanceDirectWithdrawVesting( vesting,  receiver, startFrom):
```

Comment:
```suggestion
def governanceDirectWithdrawVestingFrom( vesting,  receiver, startFrom):
```
also to facilitate governance withdrawal, could you pls:
- add a function which will process in a cycle reading incomplete withdraw events
or  alternatively
- add a function which reads and calculates iterations 
prior to calling the function

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/0a232d703e2b2581ed28ba276a58fe252c1ce872#diff-e8e027d197cc31fcc725ad99f9a9cb66e296a71b12be6e068b0b3b335d2e3768)